app/components/ReportingRequirement/ReportDueIndicator.tsx

```diff
@@ -0,0 +1,129 @@
+import { faCalendarAlt, faCircle } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import FormBorder from "lib/theme/components/FormBorder";
+import { DateTime } from "luxon";
+import Link from "next/link";
+import { graphql, useFragment } from "react-relay";
+import { ReportDueIndicator_formChange$key } from "__generated__/ReportDueIndicator_formChange.graphql";
+
+interface Props {
+  /**
+   * The relay fragment
+   */
+  reportDueFormChange: ReportDueIndicator_formChange$key;
+  /**
+   * The title of the link to display in this component.
+   * e.g. "Quarterly Report" to display "Quarterly Report <reportIndex>"
+   */
+  reportTitle: string;
+}
+
+const ReportDueIndicator: React.FC<Props> = ({
+  reportDueFormChange,
+  reportTitle,
+}) => {
+  const formChange = useFragment(
+    graphql`
+      fragment ReportDueIndicator_formChange on FormChange {
+        id
+        reportingRequirement: asReportingRequirement {
+          reportDueDate
+          reportingRequirementIndex
+        }
+      }
+    `,
+    reportDueFormChange
+  );
+
+  const reportingRequirement = formChange?.reportingRequirement;
+  const hasValidReportDueDate = reportingRequirement?.reportDueDate;
+
+  // We return a negative value if report is overdue
+  const reportDueIn =
+    hasValidReportDueDate &&
+    DateTime.fromISO(reportingRequirement.reportDueDate).diff(
```

Comment:
I'm actually curious. `reportingRequirement.reportDueDate` contains timezone information as a `timestamptz` postgres type, but the `DateTime.now().toISODate()` might not, which might be an issue when using the app from another timezone?

Reply:
Docs are a bit unclear about defaults, but from what I understand, you have to tell luxon that you want to use the timezone in the date string. Otherwise, it will use your system's timezone: https://moment.github.io/luxon/api-docs/index.html#:~:text=opts.zone%20(,it%20specifies%20one